web-animations-1/Overview.bs

```diff
@@ -1325,16 +1336,37 @@ follows:
 
     <div class="switch">
 
-    :    If <var>animation</var>'s [=playback rate=] is &ge; 0,
-    ::   Let <var>animation</var>'s <a>hold time</a> be zero.
+    :   If <var>animation</var>'s [=playback rate=] is &ge; 0,
+    ::  Update either <var>animation</var>’s <a>hold time</a> or
+        <a>start time</a> as follows:
+        <div class="switch">
+
+        :   If |animation| has no associated <a>timeline</a> or the
+            associated <a>timeline</a> is [=monotonically increasing=],
```

Comment:
As with the play procedure, it might be worth introducing a local variable to avoid repeating this condition twice?

Reply:
FIxed.